Mathlib/Data/Set/Card.lean

```diff
@@ -1070,6 +1070,8 @@ theorem exists_union_disjoint_ncard_eq_of_even_finite [DecidableEq α] (he : Eve
   simp only [← Finset.coe_union, Finite.coe_toFinset, Finset.disjoint_coe, ncard_coe_Finset,
     and_self, hutu, hdtu, hctu]
 
+open Cardinal
```

Comment:
If you're not going to use this namespace elsewhere, you can instead do `open Cardinal in` in the theorem that uses this.

Reply:
It's for the two theorems at the end of the file, so I could see the argument for either one, do you prefer one `open` or two `open in`?

Reply:
Well, I guess one open is better.